Shared/packages/MobileScenePackagesListModel.cpp

```diff
@@ -195,7 +195,7 @@ QVariant MobileScenePackagesListModel::data(const QModelIndex& index, int role)
     return QVariant();
 
   auto it = m_packageDetails.constBegin();
-  it.operator +=(index.row());
+  std::advance(it, index.row());
```

Comment:
I recommend adding `#include <iterator>` in this file to be safe.

https://en.cppreference.com/w/cpp/iterator/advance

Reply:
yup, it's there already 😄